src/scenario/scenario_events_import_xml.c

```diff
@@ -48,13 +47,16 @@ static int xml_import_special_parse_limited_number(xml_data_attribute_t *attr, i
 static int xml_import_special_parse_min_max_number(xml_data_attribute_t *attr, int *target);
 static int xml_import_special_parse_resource(xml_data_attribute_t *attr, int *target);
 static int xml_import_special_parse_route(xml_data_attribute_t *attr, int *target);
+static int xml_import_special_parse_custom_message(xml_data_attribute_t *attr, int *target);
 
 static condition_types get_condition_type_from_element_name(const char *name);
 static action_types get_action_type_from_element_name(const char *name);
 
 static int condition_populate_parameters(scenario_condition_t *condition);
 static int action_populate_parameters(scenario_action_t *action);
 
+#define XML_TOTAL_ELEMENTS 37
```

Comment:
The define should stay up on line 27 as before, we add all defines before typedefs, static variables and stuff.

Reply:
Done.